sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLDriver.scala

```diff
@@ -65,7 +65,11 @@ private[hive] class SparkSQLDriver(val context: SQLContext = SparkSQLEnv.sqlCont
       }
       context.sparkContext.setJobDescription(substitutorCommand)
       val execution = context.sessionState.executePlan(context.sql(command).logicalPlan)
-      hiveResponse = SQLExecution.withNewExecutionId(execution) {
+      val queryExecutionName = execution.executedPlan match {
+        case _: CommandResultExec => None
```

Comment:
`sql("SHOW TABLES").collect()` will show 2 entries in the SQL UI, one for executing the command, one for collecting the command result. Do we collect the command result here?

Reply:
At present, the command sql of cli will display two entries in the ui, but the listener can only collect one command.

I now change it to not exclude the command in the cli mode, so that the listener collects two entries for the command and one for the select sql, which is consistent with the behavior of the `Dataset.collect()`.